### DISCUSSION TITLE
How do I implement `LISTEN/NOTIFY` using a native `database/sql` connection?

### DISCUSSION THREAD

**Discussion by abhijit-hota** [jackc/pgx#1301] [Q&A] [answered]
I'm using `sqlx` along with `pgx` (just as a driver) for my application. How do I implement listen/notify?

**Comment by jackc** [comment, marked as answer]
Use conn.Raw() to get a `*pgx.Conn` from a `*sql.Conn`. See https://pkg.go.dev/github.com/jackc/pgx/v4@v4.17.2/stdlib for example.

Then use the `*pgx.Conn` as you would outside of `database/sql`.

**Reply by abhijit-hota** [reply]
Got it, thanks!
